src/pretalx/orga/permissions.py

```diff
@@ -0,0 +1,31 @@
+import rules
+
+from pretalx.person.permissions import is_orga, is_reviewer
+
+rules.add_perm('orga.view_orga_area', is_orga | is_reviewer)
+rules.add_perm('orga.search_all_users', is_orga)
+rules.add_perm('orga.change_settings', is_orga)
+rules.add_perm('orga.view_submission_cards', is_orga)
+rules.add_perm('orga.edit_cfp', is_orga)
+rules.add_perm('orga.view_question', is_orga)
+rules.add_perm('orga.edit_question', is_orga)
+rules.add_perm('orga.remove_question', is_orga)
+rules.add_perm('orga.view_submission_type', is_orga)
+rules.add_perm('orga.edit_submission_type', is_orga)
+rules.add_perm('orga.remove_submission_type', is_orga)
+rules.add_perm('orga.view_mails', is_orga)
+rules.add_perm('orga.send_mails', is_orga)
+rules.add_perm('orga.purge_mails', is_orga)
+rules.add_perm('orga.view_review_dashboard', is_orga | is_reviewer)
+rules.add_perm('orga.view_reviews', is_reviewer)
+rules.add_perm('orga.remove_review', is_reviewer)
```

Comment:
This means that a reviewer can remove reviews written by a different reviewer, right? I think reviewers should only be able to delete their own reviews. Reviews by other people should only be deletable by orga, or not at all.

Reply:
Fixed.